website/src/theme/DocItem/Footer/contributors.module.scss

```diff
@@ -0,0 +1,63 @@
+.contributors {
+    margin-block: var(--ifm-spacing-vertical);
+}
+
+.wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+
+    li{
+        list-style: none;
+    }
+}
+
+.unleash{
+    position: relative;
+    display: block;
+}
+
+.unleash:before{
+    display: block;
+    position: absolute;
+    z-index: 2;
+    content: '';
+    top: -5px;
+    right: -5px;
+    width: 30px;
+    height: 30px;
+    background-image: url(/img/logo.svg);
+}
+
+.contributor {
+    margin: 5px 10px 5px 0;
+    display: flex;
+    gap: 10px;
+
+    img{
+        border-radius: 100%;
```

Comment:
Any reason why we don't set this on the parent `a` tag instead? Would probably help the keyboard focus outline, at least.

Reply:
It doesn't work, it would only work if the image was a background

Reply:
What if you set `overflow: hidden`?